dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsConnector.java

```diff
@@ -61,6 +64,8 @@
 import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.jdbc.core.ResultSetExtractor;
 
+import static com.google.edwmigration.dumper.application.dumper.connector.teradata.TeradataLogsConnector.TeradataAssessmentLogsJdbcTask.ASSESSMENT_DEF_LOG_TABLE;
```

Comment:
You have a static import for a variable that you define in this file?

Reply:
Yes, I think it works this way, because this constant is defined in inner static class (that is basically the same as normal java class), that's why if you use it outside of this class you have to import it.

I agree that it looks weird, but instead I can either use it with class prefix (`TeradataAssessmentLogsJdbcTask.ASSESSMENT_DEF_LOG_TABLE`) or define in upper class `TeradataLogsConnector` (same as `DEF_LOG_TABLE` is defined).

So I've decided to move it inside `TeradataAssessmentLogsJdbcTask`, in order usage looks better and imports are usually hidden.